app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationInfo.jsx

```diff
@@ -299,6 +301,7 @@ export const IntegrationInfo = (props) => {
             onResetProjectIntegration={onResetProjectIntegration}
             isAbleToClick={isAbleToClick}
             availableProjectIntegrations={availableProjectIntegrations}
+            isAvailableLeastOneIntegration={isAvailableLeastOneIntegration}
             withButton
```

Comment:
I guess you can set `withButton={isAvailableLeastOneIntegration}` and that's it without passing the new props